mcs/class/System/Mono.Net.Security/SystemCertificateValidator.cs

```diff
@@ -45,7 +45,8 @@ static SystemCertificateValidator ()
 #elif MONODROID
 			is_macosx = false;
 #else
-			is_macosx = System.IO.File.Exists (OSX509Certificates.SecurityLibrary);
+			// make it work on wine too.
+			is_macosx = Environment.OSVersion.Platform == PlatformID.MacOSX;
```

Comment:
In my application I have to use reflection to force the is_macosx field to false when running on wine (when `Environment.OSVersion.Platform == PlatformID.Win32NT`)